cpp/log/cert.cc

```diff
@@ -546,12 +566,314 @@ Cert::Status Cert::ExtensionStructure(int extension_nid,
       LOG(WARNING) << "Corrupt extension data";
       LOG_OPENSSL_ERRORS(WARNING);
     }
+
     return FALSE;
   }
 
   return TRUE;
 }
 
+bool Cert::IsRedactedHost(const std::string& hostname) {
```

Comment:
Don't need the `std::` on string (there should be a `using std::string` above, but if not then feel free to add one.)